### PR TITLE
Dependency exclusion

### PR DESCRIPTION
**Pull request by filip-filipovic**
Added logging of the excluded dependencies. Also changed the excluded_deps file so that inject and inject-java projects are included in the report

### CODE REVIEW

**Comment by MartinBalin** [comment]
Can it exclude also inject-kotlin and inject-kotlin-test?

**Comment by filip-filipovic** [comment]
They should be excluded now

File: src/resources/excluded_deps.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ benchmark
 inject-groovy
 inject-kotlin
 inject-java-test
-inject-groovy-test
+inject-groovy
```
Comment (MartinBalin) [inline src/resources/excluded_deps.txt]: inject-groovy seems to be there 2x

Comment (filip-filipovic) [reply src/resources/excluded_deps.txt]: Right, mb, it should be fixed now

**Review by MartinBalin — APPROVED** [review]
Cannot speak much for Groovy code but I like the change and exclude file. Cedric has of course the final word...

**Review by melix — APPROVED** [review]

**Pull request merged**